Milliner/README.md

```diff
@@ -0,0 +1,71 @@
+# Milliner
+
+## Introduction
+
+Microservice that converts Drupal entities into Fedora resources.
+
+## Installation
+
+Milliner requires a functioning [Drupal 8](https://www.drupal.org/docs/8/install) installation with the [jsonld](http://github.com/Islandora-CLAW/jsonld) module enabled.
+It also requires a functioning [Fedora](http://fedorarepository.org/) repository.
+You will also need an SQL database as needed for [Gemini](../Gemini)
+
+- Clone this repository somewhere in your web root.
+- Install `composer`.  [Install instructions here.][4]
+- `$ cd /path/to/Milliner` and run `$ composer install`
+- Then either
+  - For production, configure your web server appropriately (e.g. add a VirtualHost for Houdini in Apache) OR
+  - For development, run the PHP built-in webserver `$ php -S localhost:8888 -t src` from Houdini root.
+
+## Configuration
+
+In the [config file](cfg/cfg.php) you will need to set the `fedora base url` entry to point to your Fedora installation.
+You will also need to set up the `drupal base url` entry to point to your Drupal 8 installation.
+The SQL db can be configured by following [Gemini's instructions](../Gemini).
+
+## Usage
+
+Milliner sets up a single endpoint, `/metadata/{drupal_path}`, which accepts POST, PUT, and DELETE requests.
+If requests are successful, they return the response from the Fedora server.  If the Drupal entity cannot be requested,
+the error response from Drupal is returned.  If there is an exception thrown during execution, a 500 response is returned
+with the exception's message.
+
+
+### POST
+
+This retrieves a jsonld representation of the specified Drupal entity and inserts it in Fedora.
+
+For example, suppose you create an entity at `http://localhost:8000/fedora_resource/`.  If running the PHP built-in server command described in the Installation section:
+```
+$ curl -X "POST" "localhost:8888/metadata/fedora_resource/1"
```

Comment:
Hm, that's weird. When I fired up the VM, it just offered the choices "Page" or "Article" to create. Maybe I did something wrong. 

Anyways, I'm sorry to be so thick about this, but what is the difference between `localhost:8888/metadata/fedora_resource/1` and `localhost:8888/metadata/other_fedora_resource/1` and `localhost:8888/metadata/fedora_resource/2`? If I do five `POST`s to `localhost:8888/metadata/fedora_resource/` I'll be at `localhost:8888/metadata/fedora_resource/5`, right? Will I have created five separate resources in Fedora, all of them copied from the same Drupal resource `http://localhost:8000/fedora_resource/`?

Or is it that `localhost:8888/metadata/fedora_resource/1` is the endpoint associated with a Drupal resource `http://localhost:8000/fedora_resource/1`? In that case, should it say above "suppose you create an entity at `http://localhost:8000/fedora_resource/1`" (note that I added the `1`)